testing/rclone/APKBUILD

```
@@ -0,0 +1,98 @@
+# Contributor: tcely <rclone+aports@tcely.33mail.com>
+# Maintainer: tcely <rclone+aports@tcely.33mail.com>
+_pkgname="rclone"
+pkgname="${_pkgname}"
+pkgver="1.48.0"
+pkgrel=0
+pkgdesc="rsync for cloud storage"
+url="https://rclone.org/"
+arch="all"
+license="MIT"
+options=""
+depends=""
+makedepends="bash git go perl"
+install=""
+subpackages=""
+
+gpg_signature_extensions="."
+gpgfingerprints="good:FBF7 37EC E9F8 AB18 604B  D2AC 9393 5E02 FF3B 54FA"
+gpgsource="${pkgname}-${pkgver}-sha256sums::https://downloads.rclone.org/v${pkgver}/SHA256SUMS"
+source="https://downloads.rclone.org/v${pkgver}/${_pkgname}-v${pkgver}.tar.gz $gpgsource"
+
+builddir="${srcdir}/${_pkgname}-v${pkgver}"
+export GOPATH="${srcdir}/go"
```

The `{}`s are useless.
There is no styleguide, but I'd recommend just having the GOPATH be `$srcdir`, for consistency with other go packages.
```suggestion
export GOPATH="$srcdir"
```

---

Why should we use a path that allows conflicts instead of one that does not?

---

Under normal (abuild) circumstances, `$GOPATH` only contains `src/$host/[...]`.
I don't see how this allows for any further conflicts than having a `go/src/$host` - both have exactly one conflict point, both of which are unlikely.
However, not having an extra path means less paths to traverse if debugging ends up required.

As such, can you elaborate on the conflicts part?

---

$srcdir has all sources and extracted archives.

It is less likely to have go directory already extracted than src. Ideally this should be distinct like pkgbasedir is.

---

I'd argue that an archive with a root `src` directory is just as broken as one with a root `go` directory, but fair enough.